<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>731</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Params</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>331</width>
          <height>641</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Edge</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="rInput">
             <property name="text">
              <string>400</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="moveLayout">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Move</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>dx</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="dxInput"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>dy</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="dyInput"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>dz</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="dzInput"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="scaleLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>X scale</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="xScaleInput">
               <property name="sliderPosition">
                <number>0</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Y scale</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="yScaleInput">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Z scale</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="zScaleInput">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="rotateLayout">
           <item>
            <layout class="QVBoxLayout" name="xrotateLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>X rotate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDial" name="xRotateInput">
               <property name="sliderPosition">
                <number>80</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="yrotateLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Y rotate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDial" name="yRotateInput">
               <property name="sliderPosition">
                <number>80</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="zrotateLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Z rotate</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDial" name="zRotateInput">
               <property name="sliderPosition">
                <number>80</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>Z Axes Mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="removeLinesCheckBox">
             <property name="text">
              <string>Remove hidden lines</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="radioButton">
             <property name="text">
              <string>Ort</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_2">
             <property name="text">
              <string>Perspective</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuProgram"/>
  </widget>
  <action name="actionAbout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
